Fixed 500 error in MultiTableQuery page

Gaurav-Punjabi opened the pull request:
Fixed the 500 error when a empty query is passed to MultiTableQuery.
This is in reference to solve issue #14298.

### Description
Fixed error 500 on MultiTableQuery page when a empty query is passed.

Please describe your pull request.

Fixes #14298 

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

williamdes commented on js/db_multi_table_query.js:
```diff
@@ -118,6 +118,12 @@ AJAX.registerOnload('db_multi_table_query.js', function () {
         var query = editor.getDoc().getValue();
+        // Verifying that the given query is not empty
+        if (query === '') {
+            PMA_ajaxShowMessage('Please enter the sql query first.', false, 'error');
```
Can you use `PMA_messages` like in this example (for translation) https://github.com/phpmyadmin/phpmyadmin/blob/4d8310432f2c46a425cd1a61c5fec3b9e82b82a7/js/db_search.js#L55

Thank you for the PR :)

Gaurav-Punjabi replied:
Yeah sure

Gaurav-Punjabi committed:
Replaced the info error with PMA_messages

Signed-off-by: Gaurav Punjabi <gaurav.bpunjabi@gmail.com>

Gaurav-Punjabi committed:
Fixed 500 error in MultTableQuery

Fixed the 500 error occured when a empty query is passed to the the MutliTableQuery.

Signed-off-by: Gaurav Punjabi <gaurav.bpunjabi@gmail.com>

williamdes reviewed (approved):
LGTM
Thank you !

Gaurav-Punjabi commented on js/messages.php:
```diff
@@ -139,6 +139,9 @@ function () {
 /* l10n: Default label for the y-Axis of Charts */
 $js_messages['strYValues'] = __('Y values');
 
+/* DB MULTI TABLE QUERY */
```
Okay no problem

Gaurav-Punjabi commented on js/messages.php:
```diff
@@ -139,6 +139,9 @@ function () {
 $js_messages['strYValues'] = __('Y values');
 
+/* DB MULTI TABLE QUERY */
+$js_messages['strEmptyQuery'] = __('Please enter the sql query first.');
```
Yeah sure. Will change it

Gaurav-Punjabi replied:
Done made the changes

Gaurav-Punjabi committed:
Fixed some minor issues

Translated the comment into Sentence case and the error message

Signed-off-by: Gaurav Punjabi <gaurav.bpunjabi@gmail.com>

mauriciofauth commented:
Merged, thanks for your contribution!
